src/js/views/createDevice/steps/SecurityStep/SecurityStep.jsx

```diff
@@ -44,12 +43,17 @@ const SecurityStep = ({
   };
 
   const handleOnClickCreation = () => {
-    // TODO
+    dispatch(
+      certificatesActions.createOneClick({
+        commonName: 'Dojot - IoT',
```

Comment:
Maybe this string can be stored in a constant in `/constants ` or you can use the backstage default comonName